can't compile 1.3.0 (in framework7)

Works on 1.2.5 but not 1.3.0 I get the following errors:

/Users/locutusofpenguin/VueCl7/cordova/platforms/android/app/src/main/java/com/megster/cordova/ble/central/BLECentralPlugin.java:661: error: cannot find symbol
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
                                                        ^
  symbol:   variable Q
  location: class VERSION_CODES
/Users/locutusofpenguin/VueCl7/cordova/platforms/android/app/src/main/java/com/megster/cordova/ble/central/BLECentralPlugin.java:669: error: cannot find symbol
                        Manifest.permission.ACCESS_BACKGROUND_LOCATION
                                           ^
  symbol:   variable ACCESS_BACKGROUND_LOCATION
  location: class permission
Note: /Users/locutusofpenguin/VueCl7/cordova/platforms/android/app/src/main/java/com/megster/cordova/ble/central/BLECentralPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
2 errors

FAILURE: Build failed with an exception.

--- a/src/android/BLECentralPlugin.java
+++ b/src/android/BLECentralPlugin.java
@@ -658,7 +658,7 @@ public class BLECentralPlugin extends CordovaPlugin implements BluetoothAdapter.
             LOG.w(TAG, "Location Services are disabled");
         }
 
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+        if (Build.VERSION.SDK_INT >= 29) {                                  // (API 29) Build.VERSION_CODES.Q
             if (!PermissionHelper.hasPermission(this, Manifest.permission.ACCESS_FINE_LOCATION)) {
                 permissionCallback = callbackContext;
                 this.serviceUUIDs = serviceUUIDs;
@@ -666,7 +666,7 @@ public class BLECentralPlugin extends CordovaPlugin implements BluetoothAdapter.
 
                 String[] permissions = {
                         Manifest.permission.ACCESS_FINE_LOCATION,
-                        Manifest.permission.ACCESS_BACKGROUND_LOCATION
+                        "android.permission.ACCESS_BACKGROUND_LOCATION"     // (API 29) Manifest.permission.ACCESS_BACKGROUND_LOCATION
                 };
 
                 PermissionHelper.requestPermissions(this, REQUEST_ACCESS_LOCATION, permissions);

